Resources/Prototypes/Entities/Stations/base.yml

```
@@ -78,6 +78,21 @@
+- type: entity
+  id: BaseStationShuttlesCargoOnly
+  abstract: true
+  components:
+    - type: StationCargoShuttle
+      path: /Maps/Shuttles/cargo.yml
+    - type: GridSpawn
+      groups:
+        trade:
+          addComponents:
+            - type: ProtectedGrid
+          # If you comment out the tradestation, make sure to also set cargo.tradestation_spawns_in_ftl_map to TRUE
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)